### PR TITLE
`rake install` command is failed

### PR DESCRIPTION
**Pull request by mterada1228**
### Problems

Several file paths were changed by following PR.

- https://github.com/ruby/rdoc/commit/4211292ffe80dd4737db2450d72df404a9d55051
- https://github.com/ruby/rdoc/commit/d7bca12c13b8b3f1632d698d497e67d4ea8a88bc

Because rdoc.gemspec doesn't take in this changes, the `rake install` command is permanently failed.

#### before

```console
❯ bundle exec rake install
Running RuboCop...
Inspecting 4 files
....

4 files inspected, no offenses detected

Tip: Based on detected gems, the following RuboCop extension libraries
might be helpful:
  * rubocop-rake (https://rubygems.org/gems/rubocop-rake)

You can opt out of this message by adding the following to your config
(see
https://docs.rubocop.org/rubocop/extensions.html#extension-suggestions
for more options):
  AllCops:
    SuggestExtensions: false
rake aborted!
Running `gem build -V
/Users/mterada/dev/redDataTools/remove_dependency/rdoc/rdoc.gemspec`
failed with the following output:

WARNING:  See https://guides.rubygems.org/specification-reference/ for
help
ERROR:  While executing gem ... (Gem::InvalidSpecificationException)
    ["RI.rdoc", "lib/rdoc/alias.rb", "lib/rdoc/anon_class.rb",
"lib/rdoc/any_method.rb", "lib/rdoc/attr.rb",
"lib/rdoc/class_module.rb", "lib/rdoc/constant.rb",
"lib/rdoc/context.rb", "lib/rdoc/context/section.rb",
"lib/rdoc/extend.rb", "lib/rdoc/ghost_method.rb", "lib/rdoc/include.rb",
"lib/rdoc/meta_method.rb", "lib/rdoc/method_attr.rb",
"lib/rdoc/mixin.rb", "lib/rdoc/normal_class.rb",
"lib/rdoc/normal_module.rb", "lib/rdoc/require.rb",
"lib/rdoc/single_class.rb", "lib/rdoc/top_level.rb"] are not files

/Users/mterada/.rbenv/versions/3.1.0/bin/bundle:25:in `load'
/Users/mterada/.rbenv/versions/3.1.0/bin/bundle:25:in `<main>'
Tasks: TOP => install => build
(See full trace by running task with --trace)
```

#### after

```console
❯ bundle exec rake install
Running RuboCop...
Inspecting 4 files
....

4 files inspected, no offenses detected

Tip: Based on detected gems, the following RuboCop extension libraries might be helpful:
  * rubocop-rake (https://rubygems.org/gems/rubocop-rake)

You can opt out of this message by adding the following to your config (see https://docs.rubocop.org/rubocop/extensions.html#extension-suggestions for more options):
  AllCops:
    SuggestExtensions: false
rdoc 6.7.0 built to pkg/rdoc-6.7.0.gem.
rdoc (6.7.0) installed.
```

### CODE REVIEW

File: rdoc.gemspec [hunk 1]
```diff
@@ -46,27 +46,27 @@ RDoc includes the +rdoc+ and +ri+ tools for generating and displaying documentat
...
     "TODO.rdoc",
     "exe/rdoc",
     "exe/ri",
     "lib/rdoc.rb",
-    "lib/rdoc/alias.rb",
-    "lib/rdoc/anon_class.rb",
-    "lib/rdoc/any_method.rb",
-    "lib/rdoc/attr.rb",
-    "lib/rdoc/class_module.rb",
+    "lib/rdoc/code_object/alias.rb",
```
Comment (mterada1228) [inline rdoc.gemspec:54]: Should this list of files be sorted?
To be easy to watch diff, now it is not so.

**Comment by olleolleolle** [comment]
@mterada1228 Do you think there ought to be a test which just runs `rake install`, to be able to check for errors like this?

**Comment by mterada1228** [comment]
> @mterada1228 Do you think there ought to be a test which just runs `rake install`, to be able to check for errors like this?

I agree with it. I'm going to add a test.

**Comment by st0012** [comment]
I think running it in a CI step or creating a dedicated job for it should be enough?

**Comment by mterada1228** [comment]
@olleolleolle @st0012 

Thank you for your opinions all.
I added a step to run `bundle exec rake install` to CI.

https://github.com/ruby/rdoc/pull/1170/commits/1c2bbd3a404ba255bae99073e38d4adfaa5b607c

**Review by st0012 — APPROVED** [review]
Thank you!

File: rdoc.gemspec (hunk 1 above)
Comment (st0012) [reply rdoc.gemspec:54]: Good point! I’ll open a follow up PR for some cleanups, which will include sorting the list 👍

**Pull request merged**